thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/OnboardResource.java

```diff
@@ -47,20 +41,65 @@ public OnboardResource() {
 
   public OnboardResource(AnomalyFunctionManager anomalyFunctionManager,
       MergedAnomalyResultManager mergedAnomalyResultManager,
-      RawAnomalyResultManager rawAnomalyResultManager){
+      RawAnomalyResultManager rawAnomalyResultManager) {
     this.anomalyFunctionDAO = anomalyFunctionManager;
     this.mergedAnomalyResultDAO = mergedAnomalyResultManager;
     this.rawAnomalyResultDAO = rawAnomalyResultManager;
   }
 
-  // endpoint clone function Ids to append a name defined in nameTags
+ // Useful end points to check for function information
 
   @GET
   @Path("function/{id}")
   public AnomalyFunctionDTO getAnomalyFunction(@PathParam("id") Long id) {
     return anomalyFunctionDAO.findById(id);
   }
 
+  @POST
+  @Path("function/activate")
+  public String activateFunction(@QueryParam("functionIds") String functionIds) {
```

Comment:
The activate function is implemented in AnomalyResource.activateAnomalyFunction(). Should we modify that one?